lib/ansible/module_utils/urls.py

```diff
@@ -1285,11 +1285,7 @@ def open(self, method, url, data=None, headers=None, use_proxy=None,
             else:
                 request.add_header(header, headers[header])
 
-        urlopen_args = [request, None]
-        if sys.version_info >= (2, 6, 0):
-            # urlopen in python prior to 2.6.0 did not
-            # have a timeout parameter
-            urlopen_args.append(timeout)
+        urlopen_args = [request, None, timeout]
 
         r = urllib_request.urlopen(*urlopen_args)
```

Comment:
How about changing this simply to
```suggestion
        r = urllib_request.urlopen(request, None, timeout)
```
and leaving the above lines away? Would make it more readable.

Reply:
Yes. And the temporary variable `r` can also be removed.